server/src/resolvers/reservationResolver.ts

```diff
@@ -0,0 +1,51 @@
+import { ReservationInput } from "../models/equipment/reservationInput";
+import { ReservationRepository } from "../repositories/Equipment/ReservationRepository";
+
+const reservationRepo = new ReservationRepository();
+
+const ReservationResolvers = {
+
+  Query: {
+    reservations: async (_: any, args: { Id: number }, context: any) => {
+      return await reservationRepo.getReservations();
+    },
+
+    reservation: async (_: any, args: { Id: number }, context: any) => {
+      return await reservationRepo.getReservationById(args.Id);
+    },
+
+    reservationByUser: async (_: any, args: { Id: number }, context: any) => {
```

Comment:
We had a similar conversation about holds - IMO doing `user.reservations` is more GraphQL friendly than `reservationsByUser`.

CC: @pjw9963